frontend/packages/operator-lifecycle-manager/src/components/deprecated-operator-warnings/deprecated-operator-warnings.tsx

```diff
@@ -35,17 +59,29 @@ export const DeprecatedOperatorWarningAlert: React.FC<DeprecatedOperatorWarningP
   deprecatedPackage,
   deprecatedChannel,
   deprecatedVersion,
+  dismissible,
 }) => {
   const { t } = useTranslation();
+  const [alertVisible, setAlertVisible] = React.useState<boolean>(true);
 
   return (
-    <FormAlert className="pf-v5-u-my-md">
-      <Alert variant="warning" title={t('olm~Deprecated warnings')} aria-live="polite" isInline>
-        <div>{deprecatedPackage?.deprecation?.message}</div>
-        <div>{deprecatedChannel?.deprecation?.message}</div>
-        <div>{deprecatedVersion?.deprecation?.message}</div>
-      </Alert>
-    </FormAlert>
+    alertVisible && (
+      <FormAlert className="pf-v5-u-my-md">
+        <Alert
+          variant="warning"
+          title={t('olm~Deprecated warnings')}
```

Comment:
shouldnt this be?
```suggestion
          title={t('olm~Deprecation warnings')}
```

ccing @opayne1

Reply:
Thanks for looping me in! Deprecated is correct here to be consistent with the docs. 
See[ this snippet](https://github.com/openshift/openshift-docs/blob/main/snippets/deprecated-feature.adoc) we use to call out a deprecated feature.